db/seeds.rb

```diff
@@ -39,10 +40,42 @@ def delete_columns(path, column_headers)
   row[:start_station_id] = Station.id_by_name(row.delete(:start_station_name))
   row[:end_date_id] = BikeShareDate.seed_by_date(row.delete(:end_date))
   row[:end_station_id] = Station.id_by_name(row.delete(:end_station_name))
-  row[:zipcode_id] = Zipcode.create_zipcode(row.delete(:zip_code)).id
+  row[:zipcode_id] = Zipcode.create_zipcode(row.delete(:zip_code))
```

Comment:
It seems there's no need for zipcodes in iteration 7 or 9, so let's discuss taking this out when we meet up later.